### PR TITLE
Fix docs

### PR DESCRIPTION
**Pull request by astrojuanlu**

### CODE REVIEW

**Review by ericholscher — APPROVED** [review]

**Comment by astrojuanlu** [comment]
@humitos @stsewd What would be the recommended way to install Node.js packages with the new `build` specification? I get this error, probably because `jsdoc` was installed in the old images:

```
sphinx.errors.SphinxError: jsdoc was not found. Install it using "npm install -g jsdoc".

Sphinx error:
jsdoc was not found. Install it using "npm install -g jsdoc".
```

File: docs/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 sphinx==3.5.1
```
Comment (stsewd) [inline docs/requirements.txt:1]: I think it should also be fine just updating sphinx to the latest version.

Comment (astrojuanlu) [reply docs/requirements.txt:1]: So far I'm trying to minimize changes, it's already more challenging than expected :)

Comment (astrojuanlu) [reply docs/requirements.txt]: Okay, hit the old CSS issues coming from old Sphinx + docutils 0.17. Upgrading Sphinx.

**Comment by astrojuanlu** [comment]
```
The conflict is caused by:
    The user requested docutils==0.17.1
    sphinx 4.2.0 depends on docutils<0.18 and >=0.14
    sphinx-tabs 3.2.0 depends on docutils~=0.16.0
```

🤬

**Comment by astrojuanlu** [comment]
This finally looks good again ✔️

File: tox.ini [hunk 2]
```diff
@@ -13,6 +13,7 @@ deps =
   sphinx1: Sphinx<2.0
   sphinx2: Sphinx<3.0
   sphinx3: Sphinx<4.0
+  {sphinx1,sphinx2,sphinx3}: docutils<0.18
```
Comment (humitos) [inline tox.ini:16]: oh! Nice trick! I didn't know this one :100:

**Pull request merged**